### ISSUE TITLE
Training embeddings results in unrecognizable characters

### ISSUE THREAD

**Issue by siotss96** [aria1th/Hypernetwork-MonkeyPatch-Extension#39]
I'm not sure if it was Automatic111 updates that caused the issue or something else but every time I try to create a new embedding with train_gamma, the preview images it reproduces during any Steps is always different from the subject. 

Example: 
Previously I was able to make a Lucy (edgerunners) embedding pretty easily and even during step 100 the preview image had her likeness down but now even at step 500 I would just get a random anime girl without Lucy's characteristics. 

anyone else have this issue?

**Comment by aria1th** [comment]
Unfortunately I cannot find direct change to training code - so I cannot directly compare between patches. But I suspect its fault of default adamW weight decay parameter, you can set it to `0` instead of `0.01`. Now it shares parameter with hypernetwork, so decay parameter's default value has changed from 0.

**Comment by siotss96** [comment]
doesn't seem to be that, do you know any other settings that may have been touched? 
I just tried again with a Lucy embedding still different.
![image](https://user-images.githubusercontent.com/122768943/214563989-209d2cb1-2a86-48d3-9343-1d58ba125f30.png)
